src/Compiler/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptor.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I noticed this PR was removing a couple of BOMs, so I ran Fred's script and it picked up a few stragglers. Hopefully the `.gitattributes` change makes this a thing of the past.